Vagrantfile

```diff
@@ -17,6 +17,80 @@ gopath_folder="/opt/gopath"
 
 cluster_ip_nodes = ""
 
+SUBNET_PREFIX          = "192.168."
```

Comment:
correct, I just brought in the commit temporarily so we can review that separately, and I can use it here.

I'll amend this when it's merged.